scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -892,13 +858,15 @@ public Builder setConnectionThreadCount(int threadCount) {
 		}
 
 		/**
-		 * Set the timeout of automatic session resumption in milliseconds.
+		 * Sets the timeout of automatic session resumption in milliseconds.
 		 * <p>
 		 * The default value is {@code null}, no automatic session resumption.
 		 * 
+		 * @param timeoutInMillis the number of milliseconds or {@code null} to disable automatic
+		 *                        session resumption
 		 * @return this builder for command chaining.
 		 */
-		public Builder setAutoResumptionTimeoutMillis(long timeoutInMillis) {
+		public Builder setAutoResumptionTimeoutMillis(Long timeoutInMillis) {
```

Comment:
I'm not sure, if this brings a pay-off. null would only be required, if either the default is changed or a already set value should be overwritten.

Reply:
you mean there should be no need to explicitly set it to `null`?